### PR TITLE
[bug](rf) fix invalid type for runtime filters when result column is const

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

File: be/src/vec/exprs/vruntimefilter_wrapper.cpp [hunk 1]
```diff
@@ -82,20 +83,43 @@ Status VRuntimeFilterWrapper::execute(VExprContext* context, Block* block, int*
         RETURN_IF_ERROR(_impl->execute(context, block, result_column_id));
         uint8_t* data = nullptr;
         const ColumnWithTypeAndName& result_column = block->get_by_position(*result_column_id);
-        if (auto* nullable = check_and_get_column<ColumnNullable>(*result_column.column)) {
+        if (is_column_const(*result_column.column)) {
+            auto const_data_column =
```
Comment (HappenLee) [inline be/src/vec/exprs/vruntimefilter_wrapper.cpp:87]: better call get_data_at（0）? simplify the code

**Comment by zhangstar333** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]

**Pull request merged**